src/System.Memory/tests/ReadOnlySpan/GcReporting.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.SpanTests
+{
+    public static class ReadOnlySpanGcReportingTests
+    {
+        [Fact]
+        public static void DelegateTest()
+        {
+            DelegateTest(100000, 10000);
+        }
+
+        public static void DelegateTest_Stress()
```

Comment:
The counts for DelegateTest are too high to run with GCStress=3. It didn't finish one iteration of that test even in a few hours. The counts for this test are more appropriate for running with GCStress=3, and it's sufficient to repro the failure in that mode. It'll have to be hooked into the stress infrastructure somehow, perhaps with an attribute to exclude certain tests from stress mode and include certain tests only for stress mode. I don't believe we have anything like that currently, but I just kept it in there for now to record the intended counts for that stress mode.